manager/manager.go

```diff
@@ -467,6 +472,10 @@ func (m *Manager) Run(parent context.Context) error {
 	api.RegisterNodeCAServer(m.localserver, localProxyNodeCAAPI)
 	api.RegisterResourceAllocatorServer(m.localserver, localProxyResourceAPI)
 	api.RegisterLogBrokerServer(m.localserver, localProxyLogBrokerAPI)
+	grpc_prometheus.Register(m.localserver)
+
+	// This allows to measure latency distribution.
+	grpc_prometheus.EnableHandlingTimeHistogram()
```

Comment:
Moving this to an `init` function may address the data race detected in CI.

Since it is a global setting, perhaps we should move this to the code that embeds swarmkit (such as swarmd), so swarmkit doesn't fight other code over this setting.